docs/gen-ai/gen-ai-agent-spans.md

```diff
@@ -43,6 +44,101 @@ Semantic conventions for individual GenAI systems and frameworks MAY specify dif
 <!-- markdownlint-capture -->
 <!-- markdownlint-disable -->
 
+| Attribute  | Type | Description  | Examples  | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability |
+|---|---|---|---|---|---|
+| [`gen_ai.operation.name`](/docs/attributes-registry/gen-ai.md) | string | The name of the operation being performed. [1] | `chat`; `text_completion`; `embeddings` | `Required` | ![Development](https://img.shields.io/badge/-development-blue) |
+| [`gen_ai.system`](/docs/attributes-registry/gen-ai.md) | string | The Generative AI product as identified by the client or server instrumentation. [2] | `openai` | `Required` | ![Development](https://img.shields.io/badge/-development-blue) |
+| [`gen_ai.agent.description`](/docs/attributes-registry/gen-ai.md) | string | Free-form description of the GenAI agent provided by the application. | `Helps with math problems`; `Generates fiction stories` | `Conditionally Required` If provided by the application. | ![Development](https://img.shields.io/badge/-development-blue) |
```

Comment:
is there any reason we don't have `gen_ai.agent.id` on create agent span?

Reply:
After a second thought, agree with you to add this back, as after agent is created, there will be an agent id.